src/main/scala/org/constellation/consensus/EdgeProcessor.scala

```diff
@@ -257,15 +259,15 @@ object Snapshot extends StrictLogging {
   )(
     implicit dao: DAO,
     C: ContextShift[F]
-  ): F[Path] =
+  ): F[Either[Throwable, Path]] =
```

Comment:
Instead of nesting Either inside the polimorphic type `F[Either[Throwable, Path]]` you should use `EitherT[F, Throwable, Path]`

Reply:
`EitherT` is less convenient int current scenario, as it doesn't allow chain series of polymorphic types `F` i.e:
`EItherT(...).flatMap(_ => F[..])` or `EItherT(...).flatTap(_ => F[..])` it has the `EitherT.value` accessor but then there is no point of converting back and forth.

Reply:
You can chain EitherT with another EitherT. If you want to use Either inside the F, you have to anyway pass it through the whole chain and using EitherT you get it for free

Reply:
I don't want to chain EitherT everywhere I want to mix in the `F[..]` types into chain